arch/arm64/kvm/Kconfig

```diff
@@ -107,4 +107,25 @@ config KVM_ARM_HYP_DEBUG_HYP_CALLS
 
 	  if unsure, say N.
 
+config KVM_ARM_HYP_DEBUG_RAMLOG
+	bool "Presense of a hypervisor ramlog"
+	depends on KVM_ARM_HYP_DEBUG_UART
+	default n
+	help
+		Hypervisor ramlog collects register dumps and other data separately from
```

Comment:
In general, tabs should be used as indentation . However, these Kconfig help messages are an exception, they are indented by adding two spaces. So the whole indentation (from the beginning of the line) is one tab + two spaces. A bit incongruous, but that appears to be the way it's done in the reset of the kernel.